corehq/apps/api/urls.py

```diff
@@ -11,35 +11,11 @@
 from django.conf.urls import *
 from django.http import HttpResponseNotFound
 from tastypie.api import Api
-from corehq.apps.api.es import XFormES
 
 
 API_LIST = (
-    ((0, 1), (
-        v0_1.CommCareUserResource,
-        v0_1.WebUserResource,
-        v0_1.CommCareCaseResource,
-        v0_1.XFormInstanceResource,
-        FixtureResource,
-        DomainMetadataResource,
-    )),
-    ((0, 2), (
-        v0_1.CommCareUserResource,
-        v0_1.WebUserResource,
-        v0_2.CommCareCaseResource,
-        v0_1.XFormInstanceResource,
-        FixtureResource,
-        DomainMetadataResource,
-    )),
     ((0, 3), (
-        v0_1.CommCareUserResource,
-        v0_1.WebUserResource,
         v0_3.CommCareCaseResource,
-        v0_3.XFormInstanceResource,
-        FixtureResource,
-        DomainMetadataResource,
-        locations.v0_1.LocationResource,
-        locations.v0_1.InternalLocationResource,
         ProductResource,
```

Comment:
I don't see any usage for this but it's for LMIS so not sure if that's needed still: https://confluence.dimagi.com/display/lmis/Read+API